### PR TITLE
creation-of-securityservice-and-more

### PR DESCRIPTION
**Pull request by EssienAbasiama**

### CODE REVIEW

File: src/main/java/com/myodoctor/finalproject/services/Implementations/StaffServices.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class StaffServices implements IStaffServices {
+
+    StaffServices staffServices;
+    IDeliveryPersonnelRepositories deliveryPersonnelRepositories;
+    DeliveryPersonnelRegistrationModel deliveryPersonnelModel;
+    Staff staff;
+    IPersonServices personServices;
+    PersonRegisterModel personModel;
```
Comment (usman1607) [inline src/main/java/com/myodoctor/finalproject/services/Implementations/StaffServices.java:24]: You don't have to inject register model that will be mapped with the form input field. You will pass it to the action method.

File: src/main/java/com/myodoctor/finalproject/models/RegisterModel/DoctorRegistrationModel.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import javax.persistence.Column;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+public class DoctorRegistrationModel {
+    private Staff staff;
+    private String biography;
+    private String qualifications;
+    private String documentURLs;
+    private Department department;
```
Comment (usman1607) [inline src/main/java/com/myodoctor/finalproject/models/RegisterModel/DoctorRegistrationModel.java:15]: Don't use entity in register model

File: src/main/java/com/myodoctor/finalproject/models/RegisterModel/PersonRegisterModel.java [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+public class PersonRegisterModel {
+
+    private int id;
+    private String username;
+    private String referenceNo;
+    private String firstName;
+    private String lastName;
+    private String middleName;
+    private Address address;
```
Comment (usman1607) [inline src/main/java/com/myodoctor/finalproject/models/RegisterModel/PersonRegisterModel.java:22]: Same here, dont use object in register model

File: src/main/java/com/myodoctor/finalproject/services/Implementations/DeliveryPersonnelServices.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    DeliveryPersonnelRegistrationModel deliveryPersonnelModel;
+    PersonServices personServices;
+
+    public DeliveryPersonnelServices(IDeliveryPersonnelRepositories deliveryPersonnelRepositories) {
+        this.deliveryPersonnelRepositories = deliveryPersonnelRepositories;
+    }
+
+    public boolean createDeliveryPersonnel(Person person) {
+        String role = "deliveryPersonnel";
+        staffServices.createStaff(role);
```
[the remark this answers is not in the document]
Comment (EssienAbasiama) [reply src/main/java/com/myodoctor/finalproject/services/Implementations/DeliveryPersonnelServices.java]: All Corrections have been Implemented correctly sir

**Pull request merged**